app/views/calendars/calendar_events/index.html.erb

```diff
@@ -19,14 +19,14 @@
       <table class="table table-striped">
         <thead>
           <tr>
-            <th>Type</th>
-            <th>Start Date</th>
-            <th>End Date</th>
+            <th><%= t('calendars.current_events.type') %></th>
+            <th><%= t('calendars.current_events.start_date') %></th>
+            <th><%= t('calendars.current_events.end_date') %></th>
             <th>Last Updated</th>
             <% if can? :manage, @calendar %>
-              <th>Status</th>
+              <th><%= t('calendars.current_events.status') %></th>
             <% end %>
-          <th>Actions</th>
+          <th><%= t('calendars.current_events.actions') %></th>
```

Comment:
Common label possibly?

Reply:
:+1: have pushed an update to use common labels